vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -94,8 +94,8 @@ switch ($windowsSKU) {
             "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.20.0",
             "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.19.0",
             "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.20.0",
-            "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:v1.0.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v1.0.0",
+            "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v1.2.2",
```

Comment:
Please keep the old ones. We should always follow below steps to upgrade versions:
1. Add new images in VHD in AgentBaker
2. Upgrade the versions in AKS
3. Remove old images in VHD in AgentBaker after the new versions are available in prod

Reply:
Due to some reasons, the current cache ones are not used in prod. So it is ok to delete them now.